be/src/formats/csv/string_converter.cpp

```diff
@@ -107,15 +119,28 @@ bool StringConverter::read_quoted_string(Column* column, const Slice& tmp_s, con
         max_size = options.type_desc->len;
     }
     size_t ext_size = new_size - old_size;
-    if (config::enable_check_string_lengths &&
-        ((ext_size > TypeDescriptor::MAX_VARCHAR_LENGTH) || (max_size > 0 && ext_size > max_size))) {
+
+    bool length_check_status = true;
+    // Hive table, not limit string length <= 1mb anymore
+    if (options.is_hive) {
+        if (max_size > 0 && ext_size > max_size) {
+            length_check_status = false;
+        }
+    } else {
+        if (config::enable_check_string_lengths &&
```

Comment:
could we use this option for hive also?

Reply:
`enable_check_string_lengths` default value is false, but we want to support large strings in dla by default.

Reply:
So we can't reuse this parameter

Reply:
now by default it's true.

Reply:
olap table needs enable_check_string_lengths=false